src/apps/chat/models.py

```diff
@@ -46,6 +66,11 @@ def get_upload_to(self, filename):
         return f"images/room/{self.id}/{filename}"
 
     user_attribute = "creator"
+    category = models.ForeignKey(
```

Comment:
With this data schema 1 Room belongs to 1 Category. 

In our case, Categories is User-specific, so 1 Room could belongs to multiple categories. 

So, you can have `Category.rooms` as ManyToMany field (maybe via additional through model like `CategoryRoom`).